### PR TITLE
Miscellaneous v5.6 Docs

### PR DESCRIPTION
**Pull request by jasonblais**
1. Turn on ExperimentalLimitClientConfig by default and remove setting https://github.com/mattermost/mattermost-server/pull/9696
2. Merge the experimental channel store https://github.com/mattermost/mattermost-server/pull/9681
3. Update 'Channel Display Mode' to 'Channel Display' https://github.com/mattermost/mattermost-webapp/pull/2113
4. Add Ukrainian to Mattermost https://github.com/mattermost/mattermost-webapp/pull/2098
5. Remove "Plugins > Configuration" page, move enabling plugins setting to the Management page https://github.com/mattermost/mattermost-webapp/pull/1920
6. Remove IE11 mobile view support https://github.com/mattermost/mattermost-webapp/pull/2075
7. Allow configuration of TLS Version/Ciphers/Curves and HSTS via Config File https://github.com/mattermost/mattermost-server/pull/9315

### CODE REVIEW

File: source/install/requirements.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ PC Web
     "Internet Explorer", "v11*"
     "Edge", "v42+"
 
-`*` IE 11 Compatibility View is not supported. 
+`*` IE 11 Compatibility View, nor mobile view, are supported.
```
Comment (jasonblais) [inline source/install/requirements.rst]: Is this grammatically correct?

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3269,6 +3268,52 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
 | This feature's ``config.json`` setting is ``"LicenseFileLocation": ""`` with string input.  |
 +---------------------------------------------------------------------------------------------+
 
+TLS Minimum Version
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: @crspeller Can you help review these docs related to the new TLS settings https://github.com/mattermost/mattermost-server/pull/9315 You can skip other changes on this PR.

The exact commit with TLS setting docs is here https://github.com/mattermost/docs/pull/2421/commits/55e574b3cc2d93f26053f9ca3eded4ddba838c12

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -3269,6 +3268,52 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
...
+| This feature's ``config.json`` setting is ``"TLSMinVer": "1.2"`` with string input. |
++-------------------------------------------------------------------------------------+
+
+Allow Secure TLS Transports Only
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: Only allow secure TLS transport, forcing the browser to request all resources via HTTPS. Learn more `here <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security>`_.
+
+**False**: No restrictions on TLS transport. Resources can be requested via HTTP or HTTPs.
+
```
Comment (wiersgallak) [inline source/administration/config-settings.rst]: ```Resources can be``` ==> ```Resources are````.  0/5 as I am not sure if this is accurate - the current wording makes me wonder if there is another setting for HTTP vs HTTPs, which if there is, should we link?

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -3269,6 +3268,52 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
...
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"TLSStrictTransport": false`` with options ``true`` and ``false`` for above settings respectively.         |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Secure TLS Transport Expiry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The time in seconds that the browser remembers a site is only to be accessed using HTTPS. After this period, a site can be accessed using HTTP unless ``TLSStrictTransport`` is set to ``true``.
+
+Defaults to two years.
```
Comment (wiersgallak) [inline source/administration/config-settings.rst]: Recommend moving ```Defaults to two years.``` to the prior line to be more consistent with where default values are listed in other settings.

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -3386,7 +3431,7 @@ SQL Settings
 Enable Public Channels Materialization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. note::
-  This setting provides a fast way for System Admins to disable public channels materialization if it causes unexpected performance degradation. The feature will be enabled permanently and this setting will be removed in Mattermost v5.6, to be released on December 16, 2018.
+  This setting provided a fast way for System Admins to disable public channels materialization if it caused unexpected performance degradation in Mattermost v5.4 or v5.5. The feature was permanent and this setting was removed in Mattermost v5.6, released on December 16, 2018.
```
Comment (wiersgallak) [inline source/administration/config-settings.rst]: This is a bit confusing to me.  Proposed revision to keep consistent with other removed settings:  ```*Removed in December 16, 2018 release.  The public channel materialization feature has become a permanent feature after thoroughly tested for performance.*```

**Comment by jasonblais** [comment]
@wiersgallak Good feedback! Updated

File: source/administration/config-settings.rst [hunk 6]
```diff
@@ -3269,6 +3268,50 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
...
+**False**: No restrictions on TLS transport. Resources are requested via HTTP or HTTPs.
+
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"TLSStrictTransport": false`` with options ``true`` and ``false`` for above settings respectively.         |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Secure TLS Transport Expiry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The time in seconds that the browser remembers a site is only to be accessed using HTTPS. After this period, a site can be accessed using HTTP unless ``TLSStrictTransport`` is set to ``true``. Defaults to two years.
```
Comment (crspeller) [inline source/administration/config-settings.rst]: Add the learn more about HSTS link here too.

File: source/administration/config-settings.rst [hunk 7]
```diff
@@ -3269,6 +3268,50 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
...
++-------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"TLSStrictTransportMaxAge": 63072000`` with whole number input. |
++-------------------------------------------------------------------------------------------------------------+
+
+TLS Cipher Overwrites
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set TLS ciphers overwrites to meet requirements from legacy clients which don't support modern ciphers, or to limit the types of accepted ciphers.
+
+If none specified, the Mattermost server assumes a set of currently considered secure ciphers, and allows overwrites in the edge case. See the ``ServerTLSSupportedCiphers`` variable in `/model/config.go <https://github.com/DSchalla/mattermost-server/blob/master/model/config.go>`_ for the list of ciphers considered secure.
```
Comment (crspeller) [inline source/administration/config-settings.rst]: This should point to the mattermost-server version not DSchalla's fork.

Comment (jasonblais) [reply source/administration/config-settings.rst]: Good catch, will update

**Comment by jasonblais** [comment]
@crspeller Thank you, appreciate the corrections. Have updated based on your feedback.

**Pull request merged**